tools/rapids-wheels-anaconda

```diff
@@ -3,6 +3,13 @@
 
 # Positional Arguments:
 #   1) wheel name
+#   2) package type (one of: 'cpp', 'python'). If not provided, defaults to 'python' for compatibility with older code where python was the only behavior.
+#
+# [usage]
+#
+#   # upload any wheels found in CI artifacts with names like '*wheel_python_sparkly-unicorn*.tar.gz'
+#   rapids-wheels-anaconda 'sparkly-unicorn' 'python'
+#
 
 set -exou pipefail
```

Comment:
```suggestion
set -eou pipefail
```

last request.

I usually try to omit the `-x` since it makes the logs very noisy.

I know you didn't add this flag in this PR, but I think it's still worth removing.

Reply:
Sure, fine with me! For a script this size, I'm indifferent about it 🤷🏻